src/_pytest/assertion/truncate.py

```diff
@@ -62,12 +65,18 @@ def _truncate_explanation(input_lines, max_lines=None, max_chars=None):
 
     # Append useful message to explanation
     truncated_line_count = len(input_lines) - len(truncated_explanation)
-    truncated_line_count += 1  # Account for the part-truncated final line
     msg = "...Full output truncated"
+    last_truncated = not (
+        input_lines[len(truncated_explanation) - 1] in truncated_explanation[-1]
+    )
     if truncated_line_count == 1:
-        msg += " ({} line hidden)".format(truncated_line_count)
+        msg += " ({} line hidden".format(truncated_line_count)
+    else:
+        msg += " ({} lines hidden".format(truncated_line_count)
+    if last_truncated:
+        msg += ",1 truncated)"
```

Comment:
Looks like a missing space (after comma)?

Reply:
Yes, the space was missing. I've commited 2 changes, haven't squashed yet